packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -44,8 +47,21 @@ export interface ChatCompositeProps extends BaseCompositeProps<ChatCompositeIcon
    * @defaultValue 'desktop'
    */
   formFactor?: 'desktop' | 'mobile';
+
+  /* @conditional-compile-remove(chat-reference-support) */
+  /**
+   * A callback to use as a reference to the composite.
+   * @beta
+   */
+  compositeRef?: (ref: ChatCompositeRefProps) => void;
```

Comment:
Overall this is looking good, a open question I have though is what does this do that we don't already provide with the `autofocus` property in the `ChatCompositeOptions`. We provide this as a way for contoso to tell the composite to steal focus to the send box when it is rendered.

Reply:
at the current time, nothing. 
I think the original issue was to provide a way to expand upon the boolean auto focus in a way that would allow focus / configuration for different parts within the composite. The ref does that - maybe we need a more concrete feature ask before merging?